### DISCUSSION TITLE
How about a 'server' iso without any graphical environment?

### DISCUSSION THREAD

**Discussion by cmspam** [SpiralLinux/SpiralLinux-project#505] [General]
I think that easy btrfs and snapshots pre-configured on Debian is a really great server use case, but all of your iso images have a desktop environment, from what I can tell. I would be really interested in a server version, with just a minimal terminal-based setup, but with the btrfs snapshots and rollback enhancements.

Is that something you have any interest or plan to add?

**Comment by geckolinux** [comment]
Hi there, nope, that's not within the scope of this project. Such a configuration is really not needed for a server, as a properly configured Debian Stable server will never break badly enough to need a rollback. And it's also not possible to achieve with the Calamares installer, which needs a graphical environment. If you still want to achieve something similar you can do it with the normal Debian installer:

- https://github.com/david-cortes/snapper-in-debian-guide
- https://medium.com/@inatagan/installing-debian-with-btrfs-snapper-backups-and-grub-btrfs-27212644175f
